### ISSUE TITLE
Заявки редакторам

### ISSUE THREAD

**Issue by motoprogger** [trolley813/ArchiBase#7]
**Связан ли ваш запрос с проблемой? Опишите, пожалуйста.**
У меня есть фотографии немалого числа зданий из Воркуты https://domofoto.melnix.ru/locations/3e1c7a8c-e665-4fbe-8e34-00c2c67aeb91/map и Лабытнанги https://domofoto.melnix.ru/locations/03c3bb20-db16-42f4-8658-1d4e84b46afe/map . Воркута слабо представлена в базе, а Лабытнанги - вообще никак. В то же время я не настолько искушён в архитектуре и в источниках информации о зданиях, чтобы самому становиться редактором по этим городам.

**Опишите решение, которое вы хотели бы**
1) Возможность подать заявку на внесение здания в базу для её проверки редактором
2) Возможность прикреплять фотографии при загрузке не только к уже добавленным зданиям, но и к заявкам

**Comment by trolley813** [comment]
Да, механизм заявок и правок планируется ввести, как и на старом сайте (но в более унифицированном виде, то есть, например, можно будет подавать заявки на создание/редактирование каталогов проектов и самих проектов). Ориентировочно планируется в следующей крупной версии 0.3.0
